tests/unit/routes/reminders-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:reminders', 'Unit | Route | reminders', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
```

Comment:
Also take this test out. One of these boilers will be created for you every time you use `ember g route etc`